components/button/README.md

```diff
@@ -198,7 +166,21 @@ The `d2l-button-add` element is for quickly adding new items inline (for example
 
 ## Floating Buttons [d2l-floating-buttons]
 
-See [floating buttons](https://github.com/BrightspaceUI/core/tree/main/components/button/floating-buttons.md).
+See [floating buttons](../floating-buttons).
```

Comment:
This link seems to send to a 404 page when clicking it on GitHub, I think you need the `.md` file extension at the end since there are multiple files with that name